Makefile

```
@@ -574,7 +574,9 @@ lint/install: pytest/install terraform/install terraform-docs/install cfn-lint/i
 lint/install: ec/install shellcheck/install jq/install yamllint/install
 
 install: lint/install
-install: terragrunt/install bats/install yq/install bumpversion/install docker-compose/install
+install: terragrunt/install bats/install yq/install bumpversion/install docker compose/install
```

Delete the dependency `docker-compose/install` entirely

```suggestion
install: terragrunt/install bats/install yq/install bumpversion/install
```